lua/entities/gmod_wire_expression2/core/player.lua

```diff
@@ -958,6 +985,70 @@ E2Lib.registerEvent("playerChangedTeam", {
 	{ "NewTeam", "n" }
 })
 
+--[[--------------------------------------------------------------------------------------------]]--
+
+__e2setcost(1)
+
+
```

Comment:
Would prefer if you removed the lines entirely when you removed the comments

Reply:
goddamn it

Reply:
DONE